happy-travel/kanazawa
# Add offline booking notification client request

Issue: https://github.com/happy-travel/agent-app-project/issues/1594
Test cases: 
 1) Book by offline by 3 or more days to deadline (best case)
 2) Wait for kanazawa proceeds each hours 
 3) Check if notifications sended
